client/packages/common/src/ui/components/footer/ActionsFooter.tsx

```diff
@@ -3,18 +3,18 @@ import {
   Stack,
   useTranslation,
   Typography,
-  Button,
+  FlatButton,
 } from '@openmsupply-client/common';
 
-interface ActionFooter {
+export interface ActionButtonFooter {
```

Comment:
I think `FooterAction`... or even just `Action` if it's not already taken as a name, might be better - this sounds like you are defining the props for the footer, rather than one of the actions in the footer?